uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -35,15 +35,16 @@ define([
       this.absenceTypes = [];
       this.calendar = {};
       this.contactName = null;
-      this.submitting = false;
       this.errors = [];
       this.managedContacts = [];
       this.requestDayTypes = [];
       this.selectedAbsenceType = {};
+      this.statusNames = sharedSettings.statusNames;
       this.period = {};
       this.postContactSelection = false; // flag to track if user is selected for enabling UI
       this.requestStatuses = {};
-      this.statusBeforeEdit = {};
+      this.managerDropDownStatus = null;
+      this.submitting = false;
```

Comment:
are you rearranging the properties in alphabetical order? if so, not all of them are in order